apps/_pomodoro_app/realtime/store/store.js

```diff
@@ -0,0 +1,3 @@
+const Store = require('../application/Store')
```

Comment:
it's used once. It's better to remove it

Reply:
Yes, moving the code of store layer actually to the store layer would make it easier to read

Reply:
Fixed